controllers/subscriptions.go

```diff
@@ -141,6 +144,16 @@ func Index() func(http.ResponseWriter, *http.Request) {
 		validAccNum := !(accNum == "" || accNum == "-1")
 		validOrgId := !(orgId == "" || orgId == "-1")
 
+		include_list := req.URL.Query().Get("include_bundles")
+		if include_list != "" {
+			include_filter = strings.Split(include_list, ",")
+		} else {
+			exclude_list := req.URL.Query().Get("exclude_bundles")
+			if exclude_list != "" {
+				exclude_filter = strings.Split(exclude_list, ",")
+			}
+		}
+
```

Comment:
Just an idea, but I wonder if it would clean this function up a bit to abstract this out (and remove the initialization above) and have something like:
```suggestion
		include_filter := filtersFromParams(req, "include_bundles")
		exclude_filter := filtersFromParams(req, "exclude_bundles")
```

And then define a helper:
```go
func filtersFromParams(req *http.Request, filterName string) []string{
       var filter [] string
       list := req.URL.Query().Get(filterName)
       if list != "" {
               filter = strings.Split(list, ",")
       }
       return filter
}
```

Reply:
I like this! Cleaner. Setting that up now!